purchase_package_qty/models/purchase_order_line.py

```diff
@@ -52,7 +53,7 @@ def _compute_amount(self):
                 'price_subtotal': taxes['total_excluded'],
             })
 
-    @api.model
+    @api.multi
```

Comment:
I guess this needs to be `multi` because of the `self.product_id.seller_ids` right?
It'd be better if we add `self.ensure_one()` as well. It makes the code more readable because that way you easily know you can only call this method using a singleton recordset.

Try to keep it in mind for the next migrations ;) this one we can leave it as it is